docs/DeveloperGuide.md

```diff
@@ -788,37 +802,70 @@ testers are expected to do more *exploratory* testing.
 
 1. Deliverables, meetings and contacts are saved automatically to ./data/.
 
-   On normal usage, 3 JSON files are created:
-      * `contactbook.json`
-      * `meetingbook.json`
-      * `deliverablebook.json`
-
+   On normal usage, 3 JSON files are created / saved - `contactbook.json`, `meetingbook.json` and `deliverablebook.json`.
    All 3 files contain information stored by the user from their respective modes.
 
    On first starting the program, a file is only created if the user inputs a command specific to that mode.
 
    1. Prerequisites: Very first time using the app.
 
-   1. Test case: Starting and close the app immediately.<br>
-      Expected: No JSON files created.
+   1. Test case: Start and close the app immediately.<br>
+      Expected: The 3 JSON files are not created.
 
-   1. Test case: Start the app. Switch to deliverable mode. Add a deliverable. Close the app.
-      Expected: Only `deliverablebook.json` created.
+   1. Test case: Start the app. Switch to deliverable mode. Add a deliverable. Close the app.<br>
+      Expected: Of the 3 JSON files, only `deliverablebook.json` created.
 
 1. Dealing with missing/corrupted data files
 
-   1. Test case: Delete `deliverablebook.json` file and start the jar file again<br>
-      Expected: Data file should re-initialise a list of sample deliverables
+   1. Test case: Delete `deliverablebook.json` file. Start the app. Switch to deliverable mode. Enter `list`. Close the app.<br>
+      Expected: `deliverablebook.json` should re-initialise a list of sample deliverables.
 
-   1. Test case: Corrupt `deliverablebook.json` under ./data/. The easiest way is to add - to a saved deliverable's milestone.
-      Expected: Similar to previous.
+   1. Test case: Corrupt `deliverablebook.json` under ./data/. The easiest way is to add - to a saved deliverable's milestone.<br>
```

Comment:
Do you mean: "The easiest way is to add a dash (`-`) to a saved deliverable's milestone."?

Also, not very sure if "easiest way" may imply other ways. May want to skip to just "Add a dash (`-`) to a saved..."